python/lsst/sdm/schemas/hsc.yaml

```diff
@@ -5392,6 +5392,31 @@ tables:
     datatype: boolean
     description: Flag set for failures with an unexpected or unknown cause for the multiband Sersic model.
     fits:tunit:
+  - name: g_epoch
+    "@id": "#Object.g_epoch"
+    datatype: double
+    description: Mean epoch of the object in the g-band coadd
+    fits:tunit:
```

Comment:
I don't know what the right value is here, but I bet some combination of @erykoff (who would know exactly what's in the epoch property maps; TAI MJD?) and @isullivan (who I'd guess has some experience with date units in `sdm_schemas`) could help you figure it out.

Reply:
Similar fields in the APDB use `fits:tunit: d`, but also specify more details in the `description` field. For example, `in MJD TAI`.

Reply:
The epoch property maps are based on:
```
date = row.getVisitInfo().getDate()
return date.get(date.MJD)
```